gui/default/index.html

```diff
@@ -344,40 +344,132 @@ <h3 class="panel-title">
+          </div>
+
+          <hr class="divider" />
+
+          <div class="panel panel-default">
+            <button class="btn panel-heading" data-toggle="collapse" data-parent="#login-options" data-target="#login-option-webauthn" aria-expanded="{{isDefaultLoginOption('webauthn') ? 'true' : 'false'}}">
+              <h4 class="panel-title">
+                <div class="panel-title-text" translate>WebAuthn (Passkey)</div>
+              </h4>
+            </button>
+            <div id="login-option-webauthn" class="panel-collapse collapse {{isDefaultLoginOption('webauthn') ? 'in' : ''}}">
+              <div class="panel-body">
+                <label class="stay-logged-in">
+                  <input type="checkbox" ng-model="login.stayLoggedIn" ng-disabled="login.inProgress">&nbsp;<span translate>Stay logged in</span>
+                </label>
+
+                <div class="form-group text-right">
+                  <button type="button" class="btn btn-primary" ng-click="authenticateWebauthnFinish()" ng-disabled="login.inProgress || webauthn.errors.noCredentials || webauthn.errors.initFailed || locationDoesNotMatchWebauthnRpId()" tabindex="3">
+                    <span translate>Log in with WebAuthn</span>
+                  </button>
+                </div>
+
+                <p class="text-info" ng-if="!webauthnAvailable()" translate>
+                  Your browser does not support WebAuthn.
+                </p>
+                <div ng-if="locationDoesNotMatchWebauthnRpId()">
+                  <p class="text-info" translate>
+                    Current domain does not match WebAuthn configuration.
+                  </p>
+                  <p>
+                    <a href="{{docsURL('users/webauthn#webauthn-custom-gui-address')}}" translate>
+                      Syncthing Documentation: Customizing the GUI address
+                    </a>
+                  </p>
+                </div>
+                <p class="text-danger" ng-if="webauthn.errors.notRegistered" translate>
+                  This device or security key is not registered. Please try a different one.
+                </p>
+                <p class="text-info" ng-if="webauthn.errors.aborted" translate>
+                  Authentication aborted by user.
+                </p>
+                <p class="text-danger" ng-if="webauthn.errors.notAllowed" translate>
+                  Authentication failed, please try again.
+                </p>
+                <div ng-if="webauthn.errors.securityError">
+                  <p class="text-danger" translate>
+                    Initialization blocked by browser. Please make sure Syncthing is correctly configured.
+                  </p>
+                  <p>
+                    <a href="{{docsURL('users/webauthn#webauthn-custom-gui-address')}}" translate>
+                      Syncthing Documentation: Customizing the GUI address
+                    </a>
```

Comment:
Un-line-break, as above.

Reply:
Done in da4cd0d5c23a19869eaabcc0094d2b1bee6d7019.